src/API/Reports/Products/Controller.php

```diff
@@ -343,4 +345,83 @@ public function get_collection_params() {
 
 		return $params;
 	}
+
+	/**
+	 * Get stock status column export value.
+	 *
+	 * @param array $status Stock status from report row.
+	 * @return string
+	 */
+	protected function _get_stock_status( $status ) {
+		$statuses = wc_get_product_stock_status_options();
+
+		return isset( $statuses[ $status ] ) ? $statuses[ $status ] : '';
+	}
```

Comment:
I know this is unrelated to the changes in this PR, but just to note in case there is a quick fix. In-browser download isn't adding status correctly to Products Report. Its working fine for the emailed version.

![Screen Shot 2019-10-18 at 1 09 35 PM](https://user-images.githubusercontent.com/1922453/67056500-c39a9b80-f1a8-11e9-9d5f-95d6f57508df.png)

Reply:
Another note on statuses, this time related to changes here: Products that don't have managed inventory are labelled `N/A` for Status and Stock. The CSV is giving these values of "In Stock" and "0", which isn't correct.

![Screen Shot 2019-10-18 at 1 14 04 PM](https://user-images.githubusercontent.com/1922453/67056684-8b478d00-f1a9-11e9-895f-3d4871943c5a.png)

![Screen Shot 2019-10-18 at 1 17 40 PM](https://user-images.githubusercontent.com/1922453/67056717-b7fba480-f1a9-11e9-8bf8-aca401654bd8.png)

Reply:
Same thing with Categories Report regarding Statuses and `N/A`.

Reply:
> I know this is unrelated to the changes in this PR, but just to note in case there is a quick fix. In-browser download isn't adding status correctly to Products Report.

I'm not able to reproduce that.. even turning off inventory management in settings didn't change the in-browser export.

Reply:
> Products that don't have managed inventory are labelled N/A for Status and Stock. The CSV is giving these values of "In Stock" and "0", which isn't correct.

Fixed in f9fc0d61.

> Same thing with Categories Report regarding Statuses and `N/A`.

When viewing a single category, it's actually a product report. Also fixed in f9fc0d61.

Reply:
>I'm not able to reproduce that.. even turning off inventory management in settings didn't change the in-browser export. 

I'll reproduce and get more details so I can write a separate issue if need be.